test/common/utils/accessible-modal.e2e.spec.js

```diff
@@ -0,0 +1,45 @@
+const E2eHelpers = require('../../e2e/e2e-helpers');
+const Timeouts = require('../../e2e/timeouts.js');
+
+const modalLink = 'button[data-show="#modal-crisisline"]';
+const firstModalItem = 'a[href="tel:18002738255"]';
+const closeControl = '.va-crisis-panel.va-modal-inner button';
+const lastModalItem = 'a[href="https://www.veteranscrisisline.net/"]';
+
+module.exports = E2eHelpers.createE2eTest(
+  (client) => {
+    const { TAB, SHIFT } = client.Keys;
+
+    client
+      .url(`${E2eHelpers.baseUrl}/`);
+
+    E2eHelpers.overrideSmoothScrolling(client);
+
+    client
+      .waitForElementVisible('body', Timeouts.normal)
+      .waitForElementVisible(modalLink, Timeouts.slow)
+      .axeCheck('.main');
+
+
+    // --------------------- //
+    // --- Modal tests --- //
+    // --------------------- //
+
+    // Open modal
+    client
+      .click(modalLink)
+      .assert.isActiveElement(firstModalItem);
+
+    // Trap backward traversall
+    client
+      .keys([SHIFT, TAB, TAB, SHIFT])
```

Comment:
I've tried to find good documentation on this, but my google-fu is lacking. When the keys are in an array like this, that's for key combinations, right? (Shift + tab)

If so, why are `SHIFT` and `TAB` in there twice?

Reply:
Thanks for asking, I meant to leave a review comment on this, it is to disable the shift key, which remains activated until unset.

Reply:
I'll add a comment documenting this as well.

Reply:
And in this case, [keys](http://nightwatchjs.org/api/#keys) actually accepts a sequence of keys, which are distinct from key chords or combinations.